containers/packetfence-perl/rhel8/Dockerfile_rhel8

```diff
@@ -0,0 +1,76 @@
+FROM redhat/ubi8
+
+ARG workdir \
+    output_directory \
+    psono_nqb_rhel_subs_secret_id
+
+ENV WORKDIR=${workdir} \
+    OUTPUT_DIRECTORY=${output_directory} \
+    BASE_DIR=/usr/local/pf/lib/perl_modules
+
+ENV PERL5LIB=/root/perl5/lib/perl5:${BASE_DIR}/lib/perl5/ \
+    PKG_CONFIG_PATH=/usr/lib/pkgconfig/ \
+    CPAN_BIN_PATH="/usr/bin/cpan" \
+    CPAN_VERSION=2.36 \
+    TINI_VERSION=v0.19.0
+
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+
+WORKDIR ${WORKDIR}
+
+COPY ./addons/packetfence-perl/dependencies.csv ./addons/packetfence-perl/build_package.sh ./addons/packetfence-perl/install_cpan.py ./addons/packetfence-perl/psono.py ${WORKDIR}
+
+#Install python module
+RUN dnf -y  install python39 && python3.9 -m pip install -q  -U pip && pip install -q pynacl requests
+
+RUN --mount=type=secret,id=PSONO_API_KEY_ID \
+    --mount=type=secret,id=PSONO_API_KEY_SECRET_KEY \
+    export PSONO_API_KEY_ID=$(cat /run/secrets/PSONO_API_KEY_ID) && \
+    export PSONO_API_KEY_SECRET_KEY=$(cat /run/secrets/PSONO_API_KEY_SECRET_KEY) && \
+    export REDHAT_USERNAME=$(set -e && python3  psono.py  --api_key_id=$PSONO_API_KEY_ID --api_key_secret_key=$PSONO_API_KEY_SECRET_KEY --secret_id=${psono_nqb_rhel_subs_secret_id} --return_value=username) && \
+    export REDHAT_PASSWORD=$(set -e && python3  psono.py  --api_key_id=$PSONO_API_KEY_ID --api_key_secret_key=$PSONO_API_KEY_SECRET_KEY --secret_id=${psono_nqb_rhel_subs_secret_id} --return_value=password) && \
+    sed -i 's/\(def in_container():\)/\1\n    return False/g' /usr/lib64/python*/*-packages/rhsm/config.py && \
+    subscription-manager register --username $REDHAT_USERNAME --password $REDHAT_PASSWORD --auto-attach
+
+RUN dnf install -y openssl-devel krb5-libs mariadb-devel systemd-devel gd-devel perl-open perl-experimental \
+    perl-CPAN perl-IO-Socket-SSL perl-Net-SSLeay perl-Devel-Peek perl-CPAN-DistnameInfo && \ 
+    dnf group install -y "Development Tools"
+
+RUN subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms && \
+    dnf -y module enable virt-devel && \
+    dnf install -y libssh2-devel
+
+RUN subscription-manager unregister
+
+#RUN TestPerlConfig=$(perl -e exit); [[ ${TestPerlConfig} != "" ]] && export LC_CTYPE=en_US.UTF-8;  export LC_ALL=en_US.UTF-8
```

Comment:
If not used then remove the line or add a comment before it to explain why it's here.